pkg/account/component.go

```diff
@@ -216,16 +216,17 @@ func (c *component) UpdateAccount(ctx context.Context, user api.UpdatableAccount
 	var actions []string
 
 	// Check if GLN validity should be re-evaluated
-	var glnUpdated = keycloakb.IsUpdated(user.FirstName, oldUserKc.FirstName, user.LastName, oldUserKc.LastName)
-	if !glnUpdated && user.BusinessID.Defined {
+	var namesUpdated = keycloakb.IsUpdated(user.FirstName, oldUserKc.FirstName, user.LastName, oldUserKc.LastName)
+	var glnUpdated = false
+	if !namesUpdated && user.BusinessID.Defined {
```

Comment:
I did not drop the test because the task was to improve readability. So I did not want to change the execution path.
I think the second option does not improve the readability of this part as stated in the task definition.

Reply:
So, why can we have some cases where gln has been updated but glnUpdated is false?